common/src/main/java/io/netty/util/collection/PrimitiveCollections.java

```diff
@@ -122,9 +142,12 @@ public boolean containsValue(Object value) {
      *
      * @param <V> the value type stored in the map.
      */
-    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V>,
-            Iterable<IntObjectMap.Entry<V>> {
+    private static final class UnmodifiableIntObjectMap<V> implements IntObjectMap<V> {
         final IntObjectMap<V> map;
+        Set<Integer> keySet;
```

Comment:
@Scottmitch are you confident that the example you gave does not cause synthetic member generation for `numActiveStreams`? I would expect it would since the class is still private, but perhaps there is some hackery going on in the compiler for this case?

Regardless, I'm fine making the change ... done.